src/Evolution/Systems/Cce/Callbacks/DumpBondiSachsOnWorldtube.hpp

```diff
@@ -22,6 +22,7 @@
 #include "Evolution/Systems/Cce/OptionTags.hpp"
 #include "Evolution/Systems/Cce/Tags.hpp"
 #include "Evolution/Systems/Cce/WorldtubeModeRecorder.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
```

Comment:
We don't want to have the Cce library depend on CSW. Please forward declare the tags instead.